src/controllers/baseHandler.ts

```diff
@@ -63,6 +66,11 @@ export async function baseHandler(
       return await mentionEachUser(transformedArgument, env);
     }
 
+    case getCommandName(USER): {
+      const data = message.data?.options as Array<messageRequestDataOptions>;
+      return await userCommand(data[0].value, env);
```

Comment:
what if data is null

Reply:
Data(username) will never be null because the /user command takes an argument username which always shows name suggestions for people who exist in the server. 
**Reference Image :** 
![image](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/53934353/d2011b72-076a-46f2-99c8-80a889179abf)